sonar-core/src/main/java/org/sonar/core/issue/db/UpdateConflictResolver.java

```diff
@@ -20,20 +20,22 @@
 package org.sonar.core.issue.db;
 
 import com.google.common.annotations.VisibleForTesting;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import org.sonar.api.issue.internal.DefaultIssue;
+import org.sonar.api.utils.log.Logger;
+import org.sonar.api.utils.log.Loggers;
+import org.sonar.core.issue.DefaultIssue;
 
 /**
  * Support concurrent modifications on issues made by analysis and users at the same time
  * See https://jira.sonarsource.com/browse/SONAR-4309
+ *
+ * TODO move to compute engine
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)